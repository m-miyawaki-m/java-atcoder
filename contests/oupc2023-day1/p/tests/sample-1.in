3
4 60
1 2 3 4
4 24
1 2 3 4
7 1000000000000000000
1 2 3 4 5 6 7
