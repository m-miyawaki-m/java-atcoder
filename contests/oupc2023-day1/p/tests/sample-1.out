Yes
1 2
1 3
1 4
No
Yes
1 3
3 5
5 4
2 5
7 6
1 6
